autonomy_core/map_plan/mapper/src/voxel_mapper.cpp

```diff
@@ -7,181 +7,46 @@ namespace mapper {
 VoxelMapper::VoxelMapper(const Eigen::Vector3d& origin,
                          const Eigen::Vector3d& dim,
                          double res,
-                         int8_t val,
+                         int8_t default_val,
                          int decay_times_to_empty) {
-  origin_ = Eigen::Vector3i::Zero();
   origin_d_ = Eigen::Vector3d::Zero();
   dim_ = Eigen::Vector3i::Zero();
-  lidar_rot_ = Eigen::Affine3d::Identity();
-
   res_ = res;
-  val_default = val;
-  decay_times_to_empty = decay_times_to_empty;
-  allocate(dim, origin);
+  val_default_ = default_val;
+
+  allocate(origin, dim);
```

Comment:
Yeah, so since I was refactoring the whole class I decided to clean up the header and source file a bit (e.g. removing unused variables, removing unimplemented functions, reorganinzing the function definitions and adding more comments). In this particular case I change the order of the arguments in the allocate method to have the origin first and then the dimension because all the other functions were like this. Basically I made this change to have consistency throughtout the code. Since it is a private method, it would not affect any other code that would use the VoxelMapper class.

Reply:
Ok great, there are lots of changes (great job!) and I will do a proper review probably later today!

Reply:
Sounds good, thanks Xu. I'm also going to test the code with valgrind later today. I think I should do that before we merge this PR as well

Reply:
PS: No rush on this, just let me know when you finish.